changelogs/unreleased/gh-9266-fix-on-shutdown-and-osexit-and-cmd-expr.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
+
+* Fix on shutdown trigger + os.exit() combo when used from -e command
+  line option (gh-9266).
```

Comment:
AFAIK, we should use a paste tense in this context: https://www.tarantool.io/en/doc/latest/contributing/release_notes/

Reply:
Yes, a bugfix changelog entry usually starts from "Fixed <incorrect behavior/issue>" or "Fixed a bug when <incorrect behavior>".
I need a bit more info to suggest a good wording. What exactly was a problematic case? And what consequences it was leading to?

Reply:
Issue is if from -e command line option both on shutdown trigger is set and then `os.exit` is called then trigger is not run as expected.

Reply:
Changed to 
```
* Fixed a bug when on shutdown trigger is not run if "os.exit()" is called
  from -e command line option (gh-9266).
```